docs/VerifStrat/source/quick_start.rst

```diff
@@ -66,19 +65,21 @@ The README in the cv32e40p/sim/core directory will explain how to run other test
 Where is the RTL?
 -----------------
 
-As core-v-verif is a single repository for the verification of several different CORE-V cores, you will not find the RTL here.
-Each CORE-V core is managed in its own repository, and is automatically cloned into $CORE_V_VERIF/cores/<core_name> by the Makefiles, as needed.
-If you have successfully run the hello-world test above, then you may have noticed that the RTL was cloned to $CORE_V_VERIF/cores/cv32e40p/rtl (have a look!).
+As CORE-V-VERIF is a single repository for the verification of several different CORE-V cores, you will not find the RTL here.
+Each CORE-V core is managed in its own repository, and is automatically cloned into $CORE_V_VERIF/core-v-cores/<core_name> by the Makefiles, as needed.
+If you have successfully run the hello-world test above, then you may have noticed that the RTL was cloned to $CORE_V_VERIF/core-v-cores/cv32e40p/rtl (take a look!).
+The URL, branch and hash of the cloned RTL is obtained from $CORE_V_VERIF/cv32e40p/sim/ExternalRepos.mk (again, take a look!).
+You can override these variables on the `make` command line or by setting shell environment variables.
 
 UVM
 ---
 
 Up to this point, the discussion has been about the "core" testbench, which is intended as demonstration vehicle only.
-The primary verification environments implemented in core-v-verif are all based on the UVM.
+The primary verification environments implemented in CORE-V-VERIF are all based on the UVM.
```

Comment:
```suggestion
The primary verification environments implemented in CORE-V-VERIF are all based on UVM.
```

Reply:
Good point.  I made several changes.  Note that in some cases, just "UVM" (without "the") is appropriate.